containerized_ml_models/model_api_8/api.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import FastAPI, Request
+from pickle import load
+import numpy as np
+from uvicorn import run
+
+app = FastAPI()
+model = load(open("model.pkl", "rb"))
+
+
+@app.get("/ping")
+def ping():
+    return {"ping": "success", "model": "8"}
+
+
+@app.post("/predict")
+def predict():
+    request_data = Request.get_json()
+    data = np.array([request_data["data"]])
+    prediction = model.predict(data)
+    return {"Result": prediction.tolist()}
+
+
+if __name__ == "__main__":
+    run(app, host="0.0.0.0", port=8000)
```

Comment:
You can also try to change, to pass host and port in docker file, but not necessarily.

Reply:
I don't know how to do it